src/Components/RingdownCard.scss

```diff
@@ -20,7 +20,7 @@
   }
 }
 
-.ringdown-card__eta {
+.ringdown-card__status {
```

Comment:
Here I rename .ringdown-card__eta to .ringdown-card__status, since it's used for multiple states, not just for displaying the ETA...